gopherpolicy/pkg.go

```diff
@@ -89,34 +88,40 @@ func (v *TokenValidator) LoadPolicyFile(path string) error {
 }
 
 // CheckToken checks the validity of the request's X-Auth-Token in Keystone, and
-// returns a Token instance for checking authorization. Any errors that occur
-// during this function are deferred until Require() is called.
-func (v *TokenValidator) CheckToken(r *http.Request) *Token {
+// returns a Token instance for checking authorization.
+// The suggested HTTP status code on errors is 401 Unauthoirzed.
+func (v *TokenValidator) CheckToken(r *http.Request) (*Token, string, error) {
```

Comment:
This is function supposed to implement `CheckToken(r *http.Request) (*Token, error)` from the `Validator` interface? If yes, one of the signatures need to be adjusted.

Reply:
Ups, forgot to change that back.